pkg/reconciler/clusteringress/resources/virtual_service.go

```diff
@@ -34,20 +34,26 @@ import (
 	"github.com/knative/serving/pkg/apis/serving"
 	"github.com/knative/serving/pkg/network"
 	"github.com/knative/serving/pkg/reconciler/clusteringress/resources/names"
+	"github.com/knative/serving/pkg/resources"
 )
 
-// MakeVirtualService creates an Istio VirtualService as network programming.
-// Such VirtualService specifies which Gateways and Hosts that it applies to,
-// as well as the routing rules.
-func MakeVirtualService(ci *v1alpha1.ClusterIngress, gateways []string) *v1alpha3.VirtualService {
+// VirtualServiceNamespace gives the namespace of the child
+// VirtualServices for a given ClusterIngress.
+func VirtualServiceNamespace(_ *v1alpha1.ClusterIngress) string {
```

Comment:
do you need the arg here at all?

Reply:
yeah, this will change later when we use Namespaced variant.  Yes, the arg is unused now but I want to emphasize where it will be used for the upcoming works that will use it.